### ISSUE TITLE
Unresolved reference: MaterialDialog

### ISSUE THREAD

**Issue by Sergiyss** [afollestad/material-dialogs#1794]
I can't use your library because AS can't find the path to MaterialDialog. Pointed the way manually, did not help.

Android Studio 3.4, Gradle 3.4.0, Kotlin, Androidx

**Comment by BrettJackson1987** [comment]
I got the above error when I used "implementation 'com.afollestad.material-dialogs:**input**:3.0.0-alpha1'" WITHOUT having "implementation 'com.afollestad.material-dialogs:**core**:3.0.0-alpha1". I am not sure if they are supposed to work independent of each other, but making sure that the core library was installed took away the issue for me.

However, this issue wasn't actually happening to me until I did my latest upgrades to Android to 3.4, Kotlin to 1.3.21, etc. I then the experienced "Unresolved reference: MaterialDialog" error.

I hope this is of help to you and anyone else who may be experiencing this.
